webapp/src/main_components/Nonzone.js

```diff
@@ -15,6 +22,9 @@ export const Nonzone = ({ onClose }) => {
     const { user } = useAuth();
     const { balance } = useUserWallet(user?.uid);
 
+    const { data: bookmarks } = useLoadMyBookmarks();
```

Comment:
@bengyles you can use the changes from this file as an example for using bookmark functionalities